app/src/layouts/common/appSidebar/appSidebar.jsx

```diff
@@ -20,6 +20,12 @@ import classNames from 'classnames/bind';
 import { referenceDictionary } from 'common/utils';
 import { useIntl, defineMessages } from 'react-intl';
 import { Sidebar } from 'componentLibrary/sidebar';
+import { ServiceWithPopover } from 'layouts/common/appSidebar/helpAndService';
+import LogoHelp from 'common/img/help-inline.svg';
+import { useSelector } from 'react-redux';
```

Comment:
Please place the library imports upper than the project internals.
Please check all similar places.
Will be considered as an addition to linter as well.